contrib/hbmxml/tests/tohash.prg

```diff
@@ -53,8 +53,13 @@ STATIC FUNCTION s_NodeToHash( pNode )
                   AAdd( hHash[ mxmlGetElement( pNode ) ], hHashChild )
                ENDIF
             ELSEIF hb_mxmlGetAttrsCount( pNode ) > 0
-               hHash[ mxmlGetElement( pNode ) + "@attr" ] := hb_mxmlGetAttrs( pNode )
+                  IF Empty( hHash[ mxmlGetElement( pNode ) ] ) 
+                        hHash[ mxmlGetElement( pNode ) ] := {}
```

Comment:
I'm not talking about the logic, simply the indentation is wrong since the first iteration and got worse. It's a whitespace fix.